sites/site-utilities/src/definitions/native/html-native.definition.ts

```diff
@@ -0,0 +1,80 @@
+import { DataType } from "@microsoft/fast-tooling";
+import { WebComponentAttribute } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { voidElements } from "@microsoft/fast-tooling/dist/data-utilities/html-element";
+
+/**
+ * These definitions are dependent on a devDependency on "vscode-web-custom-data"
+ * speficially the browsers.html-data.json which defines native html elements and their attributes
+ * as used by vscode
+ */
+import * as vcodeHTMLData from "vscode-web-custom-data/data/browsers.html-data.json";
+
+const valueSets: any[] = vcodeHTMLData.valueSets;
+
+export const htmlNativeDefinitions: any = {
```

Comment:
Can you create an interface for vscode version 1.1?

Reply:
setting it to 1.1 seems to be a problem, 1 is fine if it matched but 1.1 doesn't seem to work

Reply:
That's because you didn't create your own definition, you used the toolings `WebComponentDefinition` which is strongly typed at `1`. What I'm suggesting is creating your own vscode definition for 1.1 instead of using mine, since they have differences and this will help us track that.

Reply:
I think we're talking about this which is my own definition, AFAIK
`
export interface VSCodeNativeHTMLDefinition {
    version: number;
    tags: VSCodeNativeHTMLTag[];
}
`

Reply:
![Screen Shot 2020-11-04 at 8 56 54 AM](https://user-images.githubusercontent.com/25805778/98141690-c2074880-1e7b-11eb-9f54-a28deccd44ac.png)

Reply:
I don't know why that doesn't work, possibly version of typescript we are using but I'm just casting it in the import part and I have the version set at 1.1 and it's working fine.